ui/components/ui/new-network-info/new-network-info.js

```diff
@@ -42,29 +42,29 @@ export default function NewNetworkInfo() {
     setFirstTimeUsedNetwork(providerConfig.chainId);
   };
 
-  const checkTokenDetection = useCallback(async () => {
-    try {
-      const fetchedTokenData = await fetchWithCache({
-        url: `${TOKEN_API_METASWAP_CODEFI_URL}${providerConfig.chainId}`,
-        functionName: 'getIsTokenDetectionSupported',
-      });
-      const isTokenDetectionSupported = !fetchedTokenData?.error;
-      setTokenDetectionSupported(isTokenDetectionSupported);
-      setIsLoading(false);
-    } catch {
-      // If there's any error coming from getIsTokenDetectionSupported
-      // we would like to catch this error and simply return false for the state
-      // and this will be handled in UI naturally
-      setTokenDetectionSupported(false);
-      setIsLoading(false);
-    }
-  }, [providerConfig.chainId]);
+  // const checkTokenDetection = useCallback(async () => {
+  //   try {
+  //     const fetchedTokenData = await fetchWithCache({
+  //       url: `${TOKEN_API_METASWAP_CODEFI_URL}${providerConfig.chainId}`,
+  //       functionName: 'getIsTokenDetectionSupported',
+  //     });
+  //     const isTokenDetectionSupported = !fetchedTokenData?.error;
+  //     setTokenDetectionSupported(isTokenDetectionSupported);
+  //     setIsLoading(false);
+  //   } catch {
+  //     // If there's any error coming from getIsTokenDetectionSupported
+  //     // we would like to catch this error and simply return false for the state
+  //     // and this will be handled in UI naturally
+  //     setTokenDetectionSupported(false);
+  //     setIsLoading(false);
+  //   }
+  // }, [providerConfig.chainId]);
```

Comment:
for testing purposes due to [this bug](https://github.com/MetaMask/metamask-extension/issues/22937) / https://github.com/MetaMask/metamask-extension/pull/22964